### ISSUE TITLE
Null errors

### ISSUE THREAD

**Issue by elifoster** [elifoster/GHULS#16]
The chart errors occasionally with "a is null" or "null is not an object" errors. My guess is that this would be related to users having repositories that do not have languages defined by linguist. The two users I have seen this happen for are: ProgramFOX and Strikingwolf. It might also be because these users are too pro for GHULS.

**Comment by FayeAlephNil** [comment]
I am to pro for GHULS santa, it's just why I'm sorry

**Comment by elifoster** [comment]
Damn you're probably right.

**Comment by elifoster** [comment]
I just tested it, and my proposed issue is not true.
I added a simple puts statement in the main.rb file, and running it on ProgramFOX (and still getting the error) put this: `{:JavaScript=>0.25, :Python=>27.75, :"C#"=>71.52, :Shell=>0.39, :Batchfile=>0.09}`, so it does ignore null languages properly.

**Comment by elifoster** [comment]
I believe I have narrowed down the bug. I just realized that I have an auto-coloring feature for the CLI, but not for the web. Then I decided to test running it on zmthy, because I know that they have repositories in Grace, a language not covered by github-colors. This resulted in the error. I then took a look, and found that ProgramFOX as well as Strikingwolf have repositories in Batchfile, a language which does not have a color.
